utils/src/main/scala/cromwell/pipeline/utils/ApplicationConfig.scala

```diff
@@ -8,7 +8,8 @@ sealed trait ConfigComponent
 
 final case class WebServiceConfig(interface: String, port: Int) extends ConfigComponent
 
-final case class GitLabConfig(url: String, idPath: String, token: Map[String, String], defaultFileVersion: String) extends ConfigComponent
+final case class GitLabConfig(url: String, idPath: String, token: Map[String, String], defaultFileVersion: String)
```

Comment:
Why this is introduced in your commit?

Reply:
extends ConfigComponent just jumped into another string by formatter, now this string look like it was